src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -756,11 +756,8 @@ static int do_unmap()
     return nbd;
   }
 
-  dout(20) << __func__ << ": " << "sending NBD_DISCONNECT" << dendl;
```

Comment:
Please change the the commit log message to something like below:
```
rbd-nbd: remove debug messages from do_unmap

Global context is not initialized when do_unmap is called.

Singned-off-by: ...
```
So it is clear why we had to do this.